sdk/evaluation/azure-ai-evaluation/azure/ai/evaluation/_common/constants.py

```diff
@@ -54,6 +55,7 @@ class EvaluationMetrics:
     SEXUAL = "sexual"
     PROTECTED_MATERIAL = "protected_material"
     XPIA = "xpia"
+    GROUNDEDNESS = "generic_groundedness"
```

Comment:
Why not just call it "groundedness"? The other one is called "gpt_groundedness" which reflects it's a llm based.

Reply:
The other one's name may be being changed. There's a spec to remove the "gpt_" prefix. I'll message it to you on Teams.